### DISCUSSION TITLE
SDCTools User Group Meeting Material

### DISCUSSION THREAD

**Discussion by surfing-in-the-sun** [sdcTools/UserSupport#211] [User Group]
Dear Members of the User Group,

Please, find the **presented Material and Minutes from the User Group Meeting** on 21.01.2021 [here](https://github.com/sdcTools/UserMeeting/tree/master/User%20Group%20Meeting%20%231%20-%2021.01.21).

We don't want the communication to stop there and would like to encourage you to stay in touch with us by:

1. Creating [issues](https://github.com/sdcTools/UserSupport/issues) as a first choice for any remarks on the software, e.g. report bugs and ask methodological questions.
2. Using the [Discussions Area](https://github.com/orgs/sdcTools/teams/user-group/discussions) to discuss experiences between Users and with Developers, as well as to raise general questions that are not specific to the software.

GitHub is not intended to to share sensitive or confidential information and this User Group is open to anyone, who is interested. **So please do not share real datasets here in any form**. 
If there is need to share dummy data sets, then please use the  [Discussions Area](https://github.com/orgs/sdcTools/teams/user-group/discussions) for it.

We thank you again for your time and valuable input.

Best regards, Paula

**Comment by surfing-in-the-sun** [comment]
Dear  @sdcTools/user-group,

During the User Group Meeting, there was made a request to implement the Jordi Castro Algorithm by a User. Unfortunately, we didn't have the time to pursue this remark during the meeting, but want to provide you with the information now.

The footnote about the Jordi Castro Algorithm was just updated in the [**Minutes from the User Group Meeting**](https://github.com/sdcTools/UserMeeting/blob/master/User%20Group%20Meeting%20%231%20-%2021.01.21/Minutes%20SDC%20Tools%20User%20Group%20Meeting%2021.01.21.pdf) to refer to the new [**Follow up document about the Jordi Castro Algorithm**](https://github.com/sdcTools/UserMeeting/blob/master/User%20Group%20Meeting%20%231%20-%2021.01.21/Follow%20up%20of%20SDC%20Tools%20User%20Group%20Meeting%2021.01.21.pdf) .

This document provides you with detailed information on why this algorithm will not be treated as high priority.

Best regards, Paula

**Comment by olangsrud** [comment]
Thanks for following up my request to implement Jordi Castro Algorithm even if the conclusion was negative.

It is correct that my request was primarily about the paper from Work Session on Statistical Data.

However, I also had his methods for interval protection in mind. I have, at the PSD-2018-conference, talked to Jordi Castro about this. Intervals can be computed when supressing cells. And suppressed cells can be computed when making intervals. So, the approaches are similar. As I understood, within interval protection, very fast algorithms are possible.

Best regards, Øyvind

**Comment by ppdewolf** [comment]
Actually, we already have an implementation from Juan Jose Salazar for interval protection. I think that it is the same idea that Jordi was talking to you about. 
In case of interval protection, usually the number of cells with an interval is larger than the number of cells that would be suppressed. In some sense the "pain" is then spread over more cells.
As we are not aware of any NSI that would publish intervals, we did not persue the implementation in tau-argus any further.

**Comment by olangsrud** [comment]
Thanks. One point is that the latest (Jordi Castro) algorithm for interval protection is very fast, so it becomes a fast algorithm for suppression and much faster than optimal suppression. Although, then spread is worse. Another point is that interval protection treats mixed integer correctly so that additional control (the singleton problem) is not needed. These are just some thoughts. I myself have not dealt with interval protection.
